helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -655,6 +653,14 @@ internal fun Collection<LicenseFindingCuration>.mergeLicenseFindingCurations(
     return result.values.toList()
 }
 
+/**
+ * Return a copy with sorting applied to all entry types which are to be sorted.
```

Comment:
I don't get this description, what are "entry types" and what determines if they "are to be sorted"? Maybe just say something like "sort the curation by their path"?

Reply:
Done.